pandas/core/series.py

```diff
@@ -4059,9 +4059,10 @@ def _try_cast(arr, take_fast_path):
     if issubclass(subarr.dtype.type, compat.string_types):
         # GH 16605
         # If not empty convert the data to dtype
-        if not isna(data).all():
-            data = np.array(data, dtype=dtype, copy=False)
-
-        subarr = np.array(data, dtype=object, copy=copy)
+        # GH 19853: If data is a scalar, subarr has already the result
+        if not np.isscalar(data):
```

Comment:
right but is this still an extra call here? do we need the scalar check? (and should be is_scalar anyhow if its needed)

Reply:
We need a check indeed. That was the problem from the beginning.
If it is scalar, `subarr` has already the correct result.
I'll change it to `is_scalar`